### PR TITLE
Submissions of modEvA, fuzzySim & embarcadero

### PR DESCRIPTION
**Pull request by AMBarbosa**
##
modEvA:
  
mod_evaluate: most functions of the package, e.g. AUC(), threshMeasures(), Boyce(), Dsquared(), MillerCalib(), HLfit()
pred_extrapolation: MESS()
pred_inspect: varImp()

##
fuzzySim:

occ_cleaning: cleanCoords()
env_collinearity: corSelect(), FDR(), multicol()
bias: gridRecords(), selectAbsences()
backg_sample: gridRecords(), selectAbsences()
mod_fit: multGLM()
mod_tuning: modelTrim(), stepByStep(), stepwise()
mod_ensemble: fuzzyConsensus()
mod_stack: fuzzyOverlay()
mod_multispecies: multGLM()
pred_general: getPreds()
post_processing: Fav(), fuzSim(), modOverlap(), fuzzyOverlay(), bioThreat(), fuzzyRangeChange(), rarity(), vulnerability()

##
embarcadero:

mod_fit: bart.flex(), bart.step()
mod_tuning: variable.step(), retune()
mod_evaluate: summary.bart()
pred_general: predict2.bart()
pred_inspect: partial(), spartial(), varimp(), varimp.diag(), varimp.plot()

### CODE REVIEW

**Comment by sylvainschmitt** [comment]
Dear @AMBarbosa , thanks for your submission.

**Comment by sylvainschmitt** [comment]
Please fix your metadata:

> Error in `check_cran_metadata()`: 
> -For package fuzzySim, maintainer is defined as 'A. Marcia Barbosa <ana.marcia.barbosa at gmail.com>' but should be defined > as 'A. Marcia Barbosa <ana.marcia.barbosa@gmail.com>' according to CRAN.
> -For package modEvA, maintainer is defined as 'A. Marcia Barbosa <ana.marcia.barbosa at gmail.com>' but should be defined > as 'A. Marcia Barbosa <ana.marcia.barbosa@gmail.com>' according to CRAN.

**Comment by sylvainschmitt** [comment]
Once the yaml fixed, I'm looking for reviewers: @adamlilith , @EduardoArle , @gepinillab , @hannahlowens , @jamiemkass , @sgvignali are you available? Please have a look at the [reviewing guidelines](https://github.com/sylvainschmitt/sdmverse/blob/main/REVIEWING.md)

**Comment by AMBarbosa** [comment]
I had copied that exactly from the CRAN website, but OK... Fixed as requested, though I'm not sure my new pull request went through?

**Comment by sylvainschmitt** [comment]
I recognize that my `chech_cran_metadata` function is a bit too strict and I'm trying to loosen it up as I make submissions. I'll take a look, maybe the problem is not on your end.

**Comment by sylvainschmitt** [comment]
@AMBarbosa , you were missing the `<>` around your mail in the maintainer field. I checked and they are on CRAN metadata. This is now fixed. We can proceed to review.

File: inst/extdata/packages/embarcadero.yaml
Comment (sylvainschmitt) [inline inst/extdata/packages/embarcadero.yaml:5]: @AMBarbosa , why did you add a pacakge from Carlson C.J. yourself?

File: inst/extdata/packages/wallace.yaml
Comment (sylvainschmitt) [inline inst/extdata/packages/wallace.yaml:3]: @jamiemkass & @adamlilith , as planned you can see here that submission help keeping other packages up-to-date.

**Review by sylvainschmitt — APPROVED** [review]
Great work. Your contribution can be integrated without further edits. I'll still let somebody else review the submission.  After merging your pull request, you will be added to the repository contributors. Please expect to be contacted for a review request in the future.

File: inst/extdata/packages/embarcadero.yaml
Comment (AMBarbosa) [reply inst/extdata/packages/embarcadero.yaml:5]: Because I mentioned the package in the paper, Carlson is not a co-author of the paper, and I use the package a lot in my modelling so I'm quite familiar with its functions. I'm also working on a fork of that package to implement some enhancements, and I plan to submit a pull request soon.

**Comment by sylvainschmitt** [comment]
Okay to go forward and as your submission is pretty clear I wont ask additional reviewers finally.

**Pull request merged**
